bot/code_review_bot/workflow.py

```diff
@@ -188,7 +196,10 @@ def publish(self, revision, issues, task_failures):
         # Publish reports about these issues
         with stats.timer("runtime.reports"):
             for reporter in self.reporters.values():
-                reporter.publish(issues, revision, task_failures)
+                if isinstance(reporter, PhabricatorReporter):
+                    reporter.publish(issues, revision, task_failures, link_to_doc)
+                else:
+                    reporter.publish(issues, revision, task_failures)
```

Comment:
Instead of using a different `publish` function according to the class, implement the same `publish` signature in all reporter classes.